plumber/api/importer.py

```diff
@@ -0,0 +1,573 @@
+            from_game: Whether to load from game file system
+            **options: VMT import options (see import_vmt for details)
+
+        Returns:
+            Self for method chaining
+        """
+        self._jobs.append(ImportJob(AssetType.VMT, path, from_game, **options))
+        return self
+
+    def add_vtf(
+        self, path: str, from_game: bool = True, **options
+    ) -> "ParallelImportBuilder":
+        """
+        Add a VTF import job.
+
+        Args:
+            path: Path to VTF file
+            from_game: Whether to load from game file system
+            **options: VTF import options (see import_vtf for details)
+
+        Returns:
+            Self for method chaining
+        """
+        self._jobs.append(ImportJob(AssetType.VTF, path, from_game, **options))
+        return self
+
+    def execute(self, context=None) -> None:
+        """
+        Execute all import jobs in parallel.
+
+        Args:
+            context: Blender context (uses bpy.context if None)
+
+        Raises:
+            AssetImportError: If parallel import execution fails
+        """
+        if not self._jobs:
+            return
+
+        if context is None:
+            import bpy
+
+            context = bpy.context
+
+        try:
+            # Use the new Rust API importer for better performance
+            import plumber
+
+            callbacks = _create_asset_callbacks(context)
+            threads = _get_threads_suggestion(context)
+
+            # Create API importer with material settings
+            api_importer = plumber.ApiImporter(
+                self._file_system._fs,
+                callbacks,
+                threads,
+                **self._material_settings,
+            )
+
+            # Add all jobs to the API importer
+            for job in self._jobs:
+                if job.asset_type == AssetType.VMF:
+                    # VMF jobs don't pass material options since they're already in the importer
+                    non_material_options = {
+                        k: v
+                        for k, v in job.options.items()
+                        if k not in self._material_settings
+                    }
+                    api_importer.add_vmf_job(
+                        job.path, job.from_game, **non_material_options
+                    )
+                elif job.asset_type == AssetType.MDL:
+                    # MDL jobs don't pass material options since they're already in the importer
+                    non_material_options = {
+                        k: v
+                        for k, v in job.options.items()
+                        if k not in self._material_settings
+                    }
+                    api_importer.add_mdl_job(
+                        job.path, job.from_game, **non_material_options
+                    )
+                elif job.asset_type == AssetType.VMT:
+                    api_importer.add_vmt_job(job.path, job.from_game)
```

Comment:
these should also not pass any options (vmt and vtf) since material options are in importer